Style.md

```diff
@@ -149,7 +149,7 @@ See entry for *function*.
 
 ## non-optional
 
-Hyphenated to avoid the misreading as nono-ptional.
+Hyphenated to avoid the misreading as non-optional.
```

Comment:
This is actually correct as is — it's calling out how "nonoptional" can be misread without the hyphen.

```suggestion
Hyphenated to avoid the misreading as nono-ptional.
```